frontend/javascripts/oxalis/view/action-bar/starting_job_modals.tsx

```diff
@@ -651,7 +718,8 @@ function StartJobForm(props: StartJobFormProps) {
         onChangeSelectedBoundingBox={(bBoxId) => form.setFieldsValue({ boundingBoxId: bBoxId })}
         value={form.getFieldValue("boundingBoxId")}
       />
-      <div style={{ textAlign: "center" }}>
+      {isSkeletonSelectable && <ShouldUseTreesFormItem />}
+      <div style={{ textAlign: "left" }}>
```

Comment:
Why did you change the text alignment?

Reply:
The button is also on the left for the "Run a model" tab so I guess left is consistent. I will keep it on the left now but I do not really care.